### ISSUE TITLE
core.transfers table has hadm_id not include in core.admission table

### ISSUE THREAD

**Issue by kokoko12305** [MIT-LCP/mimic-iv#84]
In document, admission table can be considered as a definition table for hadm_id.
I think all hadm_id include in admission table.
But in transfers table contain hadm_id there is not include in admission table.

query
```
select hadm_id,
    max(t1.hadm_id) as admission_id,
    max(t2.hadm_id) as transfers_id
from mimic_core.admissions t1 full join mimic_core.transfers t2 using(hadm_id)
where t1.hadm_id is null or t2.hadm_id is null
group by hadm_id
```
result
[core table issues.xlsx](https://github.com/MIT-LCP/mimic-iv/files/5684295/core.table.issues.xlsx)

Is this the intended phenomenon?
If so, please explain.

**Comment by alistairewj** [comment]
Good catch, this shouldn't happen. I think these stays are related to the MIMIC-ED component which we plan to release in the next few months. They weren't supposed to be in v0.4!

**Comment by alistairewj** [comment]
Fixed in MIMIC-IV v1.0. https://mimic-iv.mit.edu/docs/overview/changelog/#mimic-iv-v10
